upgrade/definitions/1.0.2_to_1.0.3.inc.php

```diff
@@ -11,7 +11,7 @@ function run_103() {
   // Upgrade specific variables
   $aSql[] = "UPDATE `statistics_shares` SET `valid` = '0' WHERE `valid` IS NULL;";
   $aSql[] = "UPDATE `statistics_shares` SET `pplns_valid` = '0' WHERE `pplns_valid` IS NULL;";
-  $aSql[] = "ALTER TABLE `statistics_shares` CHANGE `valid` `valid` BIGINT(20) NOT NULL DEFAULT '0', CHANGE `pplns_valid` `pplns_valid` BIGINT(20) NOT NULL DEFAULT '0';";
+  $aSql[] = "ALTER TABLE `statistics_shares` CHANGE `valid` `valid` FLOAT NOT NULL DEFAULT '0', CHANGE `invalid` `invalid` FLOAT NOT NULL DEFAULT '0', CHANGE `pplns_valid` `pplns_valid` FLOAT NOT NULL DEFAULT '0', CHANGE `pplns_invalid` `pplns_invalid` FLOAT NOT NULL DEFAULT '0';";
```

Comment:
If you change the existing schema update for 1.0.3 users that have already upgraded using development branch will not be able to (easily) upgrade to this schema. Personally, I am fine with that. Anyone being on DEV branch should know how to reset their DB version to trigger this run again. It shouldn't conflict with any existing MPOS install on `development`.